scripts/default/opensearch-dashboards/build.sh

```diff
@@ -72,6 +72,13 @@ if [ "$PLATFORM" = "windows" ]; then
     NVM_CMD="volta install node@`cat ../OpenSearch-Dashboards/.nvmrc` && volta install yarn@`jq -r '.engines.yarn' ../OpenSearch-Dashboards/package.json`"
 fi
 
+HELPER_CMD=$(jq -r '.scripts.plugin_helpers' package.json)
+if [ "$HELPER_CMD" != null ]; then
+    HELPER_STRING=plugin_helpers
+else
+    HELPER_STRING=plugin-helpers
+fi
```

Comment:
Is this a fast fix that all dashboards team needs to make a change in their package.json? Changing config during build can be dangerous.

Reply:
There is no consistency on all the dashboards on how to call plugin helper.
Not sure which repo starts using plugin-helpers and some follows but some dont.
At this point this is a quick fix but the permanent fix is to let everyone use plugin_helpers.